dataherald/api/__init__.py

```diff
@@ -49,7 +49,9 @@ def add_description(
         pass
 
     @abstractmethod
-    def add_golden_records(self, golden_records: List[GoldenRecordRequest]) -> bool:
+    def add_golden_records(
```

Comment:
also please change the `GoldenRecordRequest` to:
```
class GoldenSQLRequest(BaseModel):
    question: str
    sql_query: str
    db_alias: str
```
for more descriptive naming

Reply:
@DishenWang2023 the names are obtained from the naming convention used by the data team here: https://github.com/Dataherald/k2-monorepo/blob/main/tests/golden_records.jsonl
but I'll updated it based on what you asked